src/Local.testsettings

```diff
@@ -1,9 +1,9 @@
 ﻿<?xml version="1.0" encoding="UTF-8"?>
 <TestSettings name="Local" id="8a37c363-3fe3-4ab6-a076-e4fcf4d95a30" xmlns="http://microsoft.com/schemas/VisualStudio/TeamTest/2010">
   <Description>These are default test settings for a local test run.</Description>
-  <Deployment>
-    <DeploymentItem filename="Tester\bin\Debug\TestInput\" />
```

Comment:
Please don't remove these two `DeploymentItem` entries just yet. This will break our internal tests in VSO. We'll remove them later after we migrate the tests to GitHub.